src/main/java/Task/Task.java

```diff
@@ -0,0 +1,28 @@
+package Task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void taskDone(boolean status) {
```

Comment:
Minor coding violation! Boolean variables/methods should be named to sound like booleans, and setter methods for boolean variables should be in the following form instead:
```suggestion
    public void setDone(boolean isDone) {
```